soc/v2/raspberry_pi/Kconfig

```diff
@@ -7,12 +7,7 @@
 config SOC_FAMILY_RPI_PICO
```

Comment:
Those lines should go to the `Kconfig.soc` if they are intended to be re-used
https://github.com/zephyrproject-rtos/zephyr/blob/55b00a673f19c92af714f21e3f31ae1260f5eca4/soc/v2/raspberry_pi/Kconfig#L7-L11

And base on below lines then that seems to be the case:
https://github.com/zephyrproject-rtos/zephyr/blob/55b00a673f19c92af714f21e3f31ae1260f5eca4/soc/v2/raspberry_pi/rp2/Kconfig.soc#L11-L13

Reply:
Fixed